src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -259,7 +260,7 @@ private void initMapSelection() {
     mapListView.getSelectionModel().selectedItemProperty().addListener((observable, oldValue, newValue) -> Platform.runLater(() -> setSelectedMap(newValue)));
   }
 
-  private void setSelectedMap(MapBean newValue) {
+  protected void setSelectedMap(MapBean newValue) {
```

Comment:
If you do this to be able to test this. Make it package private instead and annotate with @VisibleForTesting

Reply:
It was changed to protected so that GenerateMapController could set the map selection after generating the map in its futures.